backend/vagrant.pm

```diff
@@ -32,7 +32,7 @@ sub new {
     my $self  = $class->SUPER::new;
     my $vars  = \%bmwqemu::vars;
     $self->{vagrant_cwd}  = File::Temp->newdir();
-    $self->{box_name}     = get_required_var('VAGRANT_BOX');
+    $self->{box_name}     = $bmwqemu::$vars{VAGRANT_BOX} || die 'Need variable \'VAGRANT_BOX\'';
     $self->{boot_timeout} = get_var('VAGRANT_BOOT_TIMEOUT') // 60;
```

Comment:
Sorry if my previous comment was not clear. Because what I meant is to apply the suggestion of avoiding `get_var` and `get_required_var` or just *any* use of testapi.pm in the backends

Reply:
How would I go about replacing `$testapi::distri->add_console` with `bmwqemu`?

Reply:
I don't know. Keep it then